internal/wasi/wasi.go

```diff
@@ -1218,7 +1244,25 @@ func (a *wasiAPI) FdRenumber(ctx wasm.Module, fd, to uint32) wasi.Errno {
 
 // FdSeek implements SnapshotPreview1.FdSeek
 func (a *wasiAPI) FdSeek(ctx wasm.Module, fd uint32, offset uint64, whence uint32, resultNewoffset uint32) wasi.Errno {
-	return wasi.ErrnoNosys // stubbed for GrainLang per #271
+	f, ok := a.opened[fd]
+	if !ok || f.file == nil {
+		return wasi.ErrnoBadf
+	}
+
+	// Note: Golang's whence and WASI's whence are compatible.
```

Comment:
what is "Golang's whence" here?

Reply:
I've clarified the comment so that it's more understandable that "Golang's whence" meant io package's Seek* constants.